grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -815,16 +816,36 @@ public GrpcServiceBuilder enableHealthCheckService(boolean enableHealthCheckServ
     }
 
     /**
-     * Sets the specified {@link GrpcStatusFunction} that maps a {@link Throwable} to a gRPC {@link Status}.
+     * Sets the specified {@link GrpcExceptionHandlerFunction} that maps a {@link Throwable}
+     * to a gRPC {@link Status}.
      *
      * <p>Note that this method and {@link #addExceptionMapping(Class, Status)} are mutually exclusive.
      */
+    @UnstableApi
+    public GrpcServiceBuilder exceptionMapping(GrpcExceptionHandlerFunction grpcExceptionHandlerFunction) {
```

Comment:
What do you think of adding a new method `exceptionHandler()` instead of overloading?